rotkehlchen/chain/ethereum/modules/uniswap/uniswap.py

```diff
@@ -806,15 +823,24 @@ def _get_trades_graph_for_address(
                     timestamp = swap['timestamp']
                     swap_token0 = swap['pair']['token0']
                     swap_token1 = swap['pair']['token1']
+
+                    try:
+                        token0_deserialized = deserialize_ethereum_address(swap_token0['id'])
+                        token1_deserialized = deserialize_ethereum_address(swap_token1['id'])
+                        from_address_deserialized = deserialize_ethereum_address(swap['sender'])
+                        to_address_deserialized = deserialize_ethereum_address(swap['to'])
+                    except DeserializationError:
+                        log.error('Failed to deserialize address in swap')
```

Comment:
as above raise remote error. here you also forgot a `continue` which would have resulted in a BOOM